crates/sui-types/src/unit_tests/messages_tests.rs

```diff
@@ -145,3 +149,75 @@ fn test_certificates() {
 
     assert!(SignatureAggregator::try_new(bad_transaction, &committee).is_err());
 }
+
```

Comment:
I would love to see negative testing here:
- making which orders we depend on clear, by checking an `AuthorityQuorumSignInfo` with a:
    + permuted `signatures` field,
    + permuted `Committee` order,
- We can receive any adversarial `AuthorityQuorumSignInfo` deserialized from the network, and provided by an adversary: what are the pathological cases we can receive, and can we make sure our verification functions return with a sentinel error, rather than panic out?
    + bitmap contents that don't match the signature vec,
    + empty bitmap / signature contents,